src/main/resources/data/minecolonies/tags/items/blacksmith_product.json

```diff
@@ -1,8 +1,9 @@
-{
-  "replace": false,
-  "values": [
-    "#forge:nuggets",
-    "#forge:ingots",
-    "minecraft:shears"
-  ]
-}
+{
+  "replace": false,
+  "values": [
+    "#forge:nuggets",
+    "#forge:ingots",
+    "minecraft:shears",
+    "minecraft:diamond"

```

Comment:
Maybe add the storage blocks tag to the blacksmith_ingredient instead?

Reply:
good idea, made that change